docs/source/guide/glossary.md

```diff
@@ -0,0 +1,59 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Glossary
+[Calibration](calibrators.md)
+: The process of choosing
+the optimal QEM method and/or the optimal parameter settings of a method for a user's specific situation
+(problem type, circuit structure, resource constraints, etc.). It is analogous to choosing a machine learning method
+and its optimal hyperparameters. (Not to be confused with "noise calibration" in the sense of
+tuning a physical device so that it better approximates some ideal property or operation.)
+
+Expectation Value
+: The expectation value of an observable $A$ on state $\rho$ is the average
+readout value when $A$ is measured on $\rho$. Mathematically, this is $\text{Tr}[A\rho]$ and usually denoted $\langle A \rangle$ (when this notation is used, the state $\rho$ that $A$ is being measured on should be clear from context). Expectation values are important 
+for near-term quantum computing because in variational quantum algorithms,
+the only role of the quantum processor is to repeatedly compute expectation values, which a classical processor then uses to perform some overall useful computational task. In Mitiq, [Executors](executors.md) are used to calculate error-mitigated expectation values.
+
+Gate Fidelity
+: A number between 0 and 1 measuring how 
+closely a particular device's (noisy) physical implementation of a gate approximates the ideal gate's action on quantum states. Mitiq implements a noise-scaling method for ZNE in which each gate of the input circuit is sampled for [unitary folding](./zne-3-options.myst#unitary-folding) with probability proportional to its infidelity (1 - fidelity), described [here](./zne-3-options.myst#folding-gates-by-fidelity) and [here](https://mitiq.readthedocs.io/en/stable/apidoc.html#mitiq.zne.scaling.folding.fold_gates_at_random) in the documentation.
```

Comment:
Will need to change the link to the `myst` file over to markdown. Looks like there is another instance on line 33.

Reply:
Sorry I thought I replied to this--changing `myst` to `md` in those instances (i.e. where there's a cross-reference to a specific subsection on another page) caused the links to stop working for me. Does it work on for you Nate? I might have messed my dependencies up on my fork.

Reply:
No problem! It worked for me locally. I've added a commit [c6b3ff7](https://github.com/unitaryfund/mitiq/pull/1647/commits/c6b3ff799c1b054bf3404c7121abb0f17918d892) changing them. Let's see if CI is happy.

Reply:
Oh nice! That's great. I hope everything else is ok re: versioning/branching. Sorry, I'm still getting used to how merging works in different situations. Eventually I'll get the workflow down. For now, let me know if you notice any other version-control issues and I can do my best to resolve them.

Reply:
It looks like my commit didn't trigger all the tests. Perhaps you can add an [empty commit](https://www.freecodecamp.org/news/how-to-push-an-empty-commit-with-git/), to try and get it to trigger the entirety of the tests?

---

[Github's docs](https://docs.github.com/en/get-started/using-git) are a good place to start, along with `git`'s [official tutorial](https://git-scm.com/docs/gittutorial). Then, of course going through the parts of the [official docs](https://git-scm.com/) as necessary is always good and provides a _ton_ of info about `git`.

Reply:
**Actually**, it may be fine: https://github.com/nickdgardner/mitiq/actions/runs/4010243321/jobs/6886525524. Looks like it triggered the tests, but they just aren't linked to this PR like they normally are. Not entirely sure why, but likely related to this being a fork. No need to add the empty commit, unless you want to.

Reply:
Ok cool, I'll do an empty commit so the tests show up here.

Thanks for the git resources! I read some of the git book, which was great, but then forgot it because I wasn't using it. The github docs seem like a good, simpler stepping-stone. I'll work through them.